magento/magento2
# Issue#20277 fixed for 2.3.x

Fixed: Downloading a file from an order where the file type custom option is no longer present results in a 404.
### Description (*)


### Fixed Issues (if relevant)

1. #20277 
2. ...

### Manual testing scenarios (*)

1. Create a product with a custom option of the type "File"
2. Create a new order and upload a file for the product as a custom option
3. Check the order and download the file, this is working.
4. Update the product and delete the custom option from the product.
5. Go back to the order and try to download the file again. No longer 404.

### Contribution checklist (*)
